test/extended/dns/dns.go

```diff
@@ -5,6 +5,7 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	ocpv1 "github.com/openshift/api/config/v1"
```

Comment:
Please group this import with the other github.com/openshift imports, in alphabetical order.